README.md

```diff
@@ -156,6 +163,7 @@ If you make refund from the Midtrans Dashboard, Refund notification is sent to M
     *   **Refund Offline**: An offline refund does not trigger request refund to midtrans it’s only refund in Magento side. You need to take action and carry out the refund manually from Midtrans dashboard.After a refund operation, the order status changes to **Closed**. This order status change is controlled by the Magento system. 
     
     The status change may not mean that the refund has carried out successfully on Midtrans side. When the transaction status in Midtrans dashboard changes to REFUND, then the refund went through successfully
+</details>
```

Comment:
Should add somewhere in this section that this extension now supports MultiShipping

Reply:
Yes, I put the info on line `111`. Should I put to that section too?
`Multishipping only support on version Midtrans Magento Plugins v2.5.3 or greater and not support in offline installment payment`

Reply:
Oh I missed that, then no need to add more. Looks good 👌